Library/Services/INfieldSurveyInterviewSimulationService.cs

```diff
@@ -29,5 +30,20 @@ public interface INfieldSurveyInterviewSimulationService
         /// <param name="surveyId">Id of the simulation survey</param>
         /// <returns>URI of the hints file</returns>
         Task<Uri> GetHintsAsync(string surveyId);
+
+        /// <summary>
+        /// Starts interview simulation.
+        /// </summary>
+        /// <param name="surveyId">The survey id</param>
+        /// <param name="simulationRequest">Parameters to start the simulations</param>
+        Task StartSimulationAsync(string surveyId, InterviewSimulation simulationRequest);
+
+        /// <summary>
+        /// Starts interview simulation.
+        /// This one allows to upload  hints and sample data directly from files.  
+        /// </summary>
+        /// <param name="surveyId">The survey id</param>
+        /// <param name="simulationRequest">Parameters to start the simulations</param>
+        Task StartSimulationAsync(string surveyId, InterviewSimulationFiles simulationRequest);
```

Comment:
It should at least return the id of the simulation survey. And probably also the error messages (if any)

Reply:
Is the idea to call from this method the BackgroundActivityController to get the activity result and return it?

Reply:
This method can also return the activity id. And the call to BackgroundActivityController could be done separately. But I don't see a service for background activity in the sdk.

Reply:
It appears that the activity status is already available. No need to call the background activity controller, just read the result from with GetActivityResultAsync